firebase-firestore/src/test/java/com/google/firebase/firestore/model/ObjectValueBuilderTest.java

```diff
@@ -74,7 +74,7 @@ public void setsNestedField() {
     builder.set(field("a.b"), fooValue);
     builder.set(field("c.d.e"), fooValue);
     ObjectValue object = builder.build();
-    assertEquals(wrapObject("a", map("b", fooValue), "c", map("d", map("e", fooValue))), object);
+    assertEquals(wrapObject("a", map("b", fooString), "c", map("d", map("e", fooString))), object);
```

Comment:
`fooString` isn't adding a whole lot of value here, and it's inconsistent with the other string literals here. Consider reverting that.

Also, why not pass the values as arguments to `map`? That seemed like a great way to explicitly compose expected values.